channel.go

```diff
@@ -123,14 +123,6 @@ func (ch *channel) recv() (messageHeader, []byte, error) {
 		return messageHeader{}, nil, err
 	}
 
-	if mh.Length > uint32(messageLengthMax) {
```

Comment:
I'd like to keep this check. While ttrpc services wouldn't internet facing, it is better to be defensive.

Reply:
I readded the code.